init-tools.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+__scriptpath=$(cd "$(dirname "$0")"; pwd -P)
+__init_tools_log=$__scriptpath/init-tools.log
+__PACKAGES_DIR=$__scriptpath/.nuget
+__TOOLRUNTIME_DIR=$__scriptpath/build_tools
+__DOTNET_PATH=$__scriptpath/.dotnet_stage0/$(uname -s)
```

Comment:
I think we want to get rid of the OS name in the stage0 path.  Can we do it here?